components/playlist/resources/playlist.tsx

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+
+// Components
+import App from './components/app'
+
+// Theme
+import BraveCoreThemeProvider from '../../common/BraveCoreThemeProvider'
+import Theme from 'brave-ui/theme/brave-default'
+import DarkTheme from 'brave-ui/theme/brave-dark'
+// Utils
+import store from './store'
+
+window.cr.define('playlist', function () {
```

Comment:
This is legacy style of communicating from c++ to JS. We should fire event instead.